mne/beamformer/_lcmv.py

```diff
@@ -352,7 +363,7 @@ def lcmv(evoked, forward, noise_cov, data_cov, reg=0.01, label=None,
 
 
 @verbose
-def lcmv_epochs(epochs, forward, noise_cov, data_cov, reg=0.01, label=None,
+def lcmv_epochs(epochs, forward, noise_cov, data_cov, reg=0.1, label=None,
```

Comment:
IIRC I tweaked the parameters in the examples to try to make them look good. Not sure what the defaults should be :(